packages/workflow/src/NodeHelpers.ts

```diff
@@ -33,7 +30,7 @@ import { get } from 'lodash';
  * @param {INodeType} nodeType
  * @returns
  */
-export function getSpecialNodeParameters(nodeType: INodeType) {
+export function getSpecialNodeParameters(nodeType: INodeType): object[] {
```

Comment:
`object` seems like a cheat around using any... maybe we should add it to ban-types 
it's included with the defaults https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md